lib/elixir/test/elixir/option_parser_test.exs

```diff
@@ -256,6 +256,12 @@ defmodule OptionParserTest do
            == {[foo: true, boo: "-"], ["-"], []}
   end
 
+  test "allow nonexistent atom`s" do
+    opts = [switches: [a: :string], allow_nonexistent_atoms: true]
+    assert OptionParser.parse(["--a", "b", "--c", "d", "--option-key-creates-atom", "f"], opts)
```

Comment:
@whatyouhide oh no, wait. Too much cases with operator on new line. If you want, I will prepare next PR that will fix code syntax in option parses tests.